base/path.jl

```diff
@@ -427,13 +444,13 @@ end
 Convert a set of paths to a normalized path by joining them together and removing
 "." and ".." entries. Equivalent to `normpath(joinpath(path, paths...))`.
 """
-normpath(a::AbstractString, b::AbstractString...) = normpath(joinpath(a,b...))
+normpath(a::AbstractString, b::AbstractString...; kwargs...) = normpath(joinpath(a,b...); kwargs...)
 
 """
-    abspath(path::AbstractString)::String
+    abspath(path::AbstractString; safe=false)::String
 
 Convert a path to an absolute path by adding the current directory if necessary.
-Also normalizes the path as in [`normpath`](@ref).
+If `normalize` is true, also normalizes the path as in [`normpath`](@ref).
```

Comment:
in the signature, the kwarg is called `safe` and not `normalize` (both in the docstring signature and function signature)

Reply:
I renamed it, but missed the docstring update.